drivers/timer/Kconfig.stm32_lptim

```diff
@@ -17,26 +17,26 @@ if STM32_LPTIM_TIMER
 choice STM32_LPTIM_CLOCK
 	prompt "LPTIM clock value configuration"
 
-config STM32_LPTIM_CLOCK_LSI
-	bool "LSI"
-	help
-	  Use LSI as LPTIM clock
-
 config STM32_LPTIM_CLOCK_LSE
 	bool "LSE"
 	help
 	  Use LSE as LPTIM clock
 
+config STM32_LPTIM_CLOCK_LSI
+	bool "LSI"
+	help
+	  Use LSI as LPTIM clock
+
 endchoice
 
 config STM32_LPTIM_CLOCK
 	int "LPTIM clock value"
-	default 32000 if STM32_LPTIM_CLOCK_LSI
 	default 32768 if STM32_LPTIM_CLOCK_LSE
+	default 32000 if STM32_LPTIM_CLOCK_LSI
 
 config STM32_LPTIM_TIMEBASE
 	hex "LPTIM AutoReload value"
-	default 0xF9FF if STM32_LPTIM_CLOCK_LSI
 	default 0xFFFF if STM32_LPTIM_CLOCK_LSE
+	default 0xF9FF if STM32_LPTIM_CLOCK_LSI
```

Comment:
Not useful but ok

Reply:
setting LSE in first position to have this as default ?

Reply:
Position only matters for choice (like STM32_LPTIM_CLOCK) actually